### PR TITLE
QUAD-112: break out sections

### PR DESCRIPTION
**Pull request by jimentwood**

### CODE REVIEW

File: help/moderation.md [hunk 1]
```diff
@@ -1,41 +1,70 @@
...
+Submissions to arXiv must be the author’s original work, and users must have the legal authority to grant the [selected license](/help/license) Users must ensure the submission does not, to the best of their knowledge, infringe upon anyone's copyright. Users should not submit articles containing verbatim comments by referees, plagiarized content, or other works barred by third party copyright. 
+
+### Excessive submission rate
+
+Articles submitted to arXiv must be of refereeable quality, and there is a practical limit to the rate at which appropriate, independent submissions can be produced by any one person. We may request that a particular author limit their submission rate if this author has a history of many submissions to inappropriate areas or of doubtful refereeability. 
 If an author has a back catalog of work to submit to arXiv we ask that they submit no more than three papers per day.
 
+## What policies guide moderation before public announcement?
```
Comment (jweiskoff) [inline help/moderation.md:42]: Should be "after public announcement"

File: help/moderation.md [hunk 2]
```diff
@@ -1,41 +1,70 @@
...
+
+Once a paper is announced it becomes part of the permanent scholarly record. While arXiv reserves the right to remove any content, arXiv will only consider requests for removal if the submitter did not have the legal right to [agree to the license](/help/license).
+
+In cases where papers that are posted are found to violate arXiv policy we may [withdraw the paper](/help/withdraw). arXiv may also indicate the concern in the Comments metadata.
+
+arXiv may reclassify already announced papers if the moderators determine there is a better categorization.
```
Comment (jweiskoff) [inline help/moderation.md:48]: Suggest "more appropriate" instead of _better_.

File: help/moderation.md [hunk 3]
```diff
@@ -1,41 +1,70 @@
...
 
+It is possible to appeal the classification of an announced article, or the decision to decline a submission. In some cases, we may require that a paper be accepted for publication in a conventional journal before an appeal is considered.
+
 If you disagree with a moderation decision, you may submit an appeal. Please carefully follow the directions below.
 
 - You can submit an appeal through our [user support portal](https://arxiv-org.atlassian.net/servicedesk/customer/portal/2) or by sending an email to to moderation@arxiv.org.
 - In your appeal please include all relevant details, including paper IDs, former correspondence, submission summaries, etc.
 - You will get an email from arXiv administrators when your appeal has been sent to our appellate moderators for consideration.
-- Please be *patient*, as moderation appeals are complicated and take some time.
+- Please be patient, as moderation appeals are complicated and take some time.
```
Comment (jweiskoff) [inline help/moderation.md:62]: Maybe add verbiage regarding moderators not being in the same geographic timezone, and therefore extended discussion may take time to resolve.

**Review by jweiskoff — CHANGES_REQUESTED** [review]
A typo and a couple minor tweaks and I think it'd be good to go.

File: help/moderation.md (hunk 1 above)
Comment (jimentwood) [reply help/moderation.md]: good catch

File: help/moderation.md (hunk 2 above)
Comment (jimentwood) [reply help/moderation.md]: done

File: help/moderation.md (hunk 3 above)
Comment (jimentwood) [reply help/moderation.md:62]: Possibly... I'm not sure we need to give any reason though. What do others think?

Comment (afromme) [reply help/moderation.md:62]: I think the simple acknowledgement that appeals take time is enough

**Review by afromme — APPROVED** [review]

File: help/moderation.md (hunk 3 above)
Comment (rebamrich) [reply help/moderation.md:62]: I agree, no reason beyond what we've stated already is really necessary.

**Review by jweiskoff — APPROVED** [review]

**Pull request merged**